prometheus/src/main/scala/filodb/prometheus/ast/Functions.scala

```diff
@@ -180,12 +180,13 @@ trait Functions extends Base with Operators with Vectors {
             rangeFunctionId, false, otherParams, instantExpression.offset.map(_.millis(timeParams.step * 1000)))
         } else {
           val rangeExpression = seriesParam.asInstanceOf[RangeExpression]
+
           PeriodicSeriesWithWindowing(
             rangeExpression.toSeriesPlan(timeParams, isRoot = false),
             timeParams.start * 1000 , timeParams.step * 1000, timeParams.end * 1000,
             rangeExpression.window.millis(timeParams.step * 1000),
             rangeFunctionId, rangeExpression.window.timeUnit == IntervalMultiple,
-            otherParams, rangeExpression.offset.map(_.millis(timeParams.step * 1000)))
+            otherParams, rangeExpression.offset.map(_.millis(timeParams.step * 1000)), rangeExpression.columnFilters)
```

Comment:
I didnt get this change. Why does AbsentFunctionMapper need columnFilters ?

Reply:
absent_over_time creates range vector with column filters in original query.